frontend/src/admin/settings/settings.controllers.js

```diff
@@ -105,7 +105,15 @@ const SECTIONS = [
                 type: "string",
                 required: true,
                 autoFocus: true
-            }
+            },
+            {
+                key: "metabot-enabled",
+                display_name: "Metabot",
+                type: "boolean",
```

Comment:
We can put some type information in defsetting, but that would be a separate change.  Also, in this particular case, even though it says "boolean" it really means "toggle".  The other types which are used in the same list of settings are things like "radio", "password", and "select", so those types are more about input type than value type.